ckanext/canada/helpers.py

```diff
@@ -538,3 +538,10 @@ def canada_check_access(package_id):
         return h.check_access('package_update', {'id': package_id})
     except NotFound:
         return False
+
+
+def user_in_organization(user):
+    u = User.get(user['name'])
+    group_ids = u.get_group_ids(group_type="organization")
+
+    return u.is_in_groups(group_ids)
```

Comment:
this checks if the user is in their own organizations?

Reply:
no, it checks groups for a user in user list against logged in user (self's) groups

Reply:
I'm reading the definition of `is_in_groups` and `get_group_ids` and I don't see any reference to the logged in user. `self` means the object the method was called against (`u` in this case)